### PR TITLE
Docs/contrib update to Style Guide

### LINKED ISSUE #809: REUSE compliance
**Is your feature request related to a problem? Please describe.**
A nice standard solving our problems with copyright headers and licenses: https://reuse.software/tutorial/

**Describe the solution you'd like**
Implement REUSE (maybe)

**Describe alternatives you've considered**
Do nothing

**Additional context**
Need to keep track of the licenses used for distribution

### PR DESCRIPTION
**Pull request by jwrober**
Closes #809
Part of #1250 

Updating to match style guide
* Standard SPDX attribution header
* Proper capitalization rules
* Image `:numref:` usage
* Cross document `:ref:` usage
* Consonant usage
* Glossary usage

### CODE REVIEW

File: docs/Contributing/release.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ These are the general steps to prepare and finalize a release:
       open a PR for this change to ``master``. This way, development builds from ``master`` will immediately
       use the version number of the next stable.
 
-#. If the release is a :strong:`release candidate` or a :strong:`stable release`, the release manager will
+#. If the release is a :strong:`release candidate` for a :strong:`stable release`, the release manager will
```
Comment (daavko) [inline docs/Contributing/release.rst:44]: Why this change? This wasn't a typo

**Review by daavko — COMMENTED** [review]
One clarification, other than that LGTM

File: docs/Contributing/release.rst (hunk 1 above)
Comment (jwrober) [reply docs/Contributing/release.rst:44]: no that change is correct. "If the release is a release candidate **for** a stable release...", the original "or" wasn't correct.
